django/db/models/sql/compiler.py

```diff
@@ -802,6 +809,35 @@ def get_related_klass_infos(klass_info, related_klass_infos):
                 )
         return related_klass_infos
 
+    def get_select_for_update_of_arguments(self):
+        """
+        Return a quoted list of arguments to the SELECT FOR UPDATE OF
+        part of the query.
+        """
+        result = []
```

Comment:
I used a list here because I build it up with `append`, and building up a tuple doesn't feel natural.

It also matches the code like in `get_distinct`: https://github.com/django/django/blob/5dbf1c4b23cda915369f4895be293369575238d0/django/db/models/sql/compiler.py#L580